controller/workitem.go

```diff
@@ -316,7 +317,13 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 		Fields: make(map[string]interface{}),
 	}
 	return application.Transactional(c.db, func(appl application.Application) error {
-		err := ConvertJSONAPIToWorkItem(ctx, appl, *ctx.Payload.Data, &wi, ctx.SpaceID)
+		//verify spaceID:
+		// To be removed once we have endpoint like - /api/space/{spaceID}/workitems
+		_, spaceLoadErr := appl.Spaces().Load(ctx, ctx.SpaceID)
+		if spaceLoadErr != nil {
+			return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("space", "string").Expected("valid space ID"))
```

Comment:
thanks @hectorj2f , done in https://github.com/fabric8-services/fabric8-wit/pull/1466/commits/656588c37614ca2aadc7aa6e4c5cb00a78736e12